simplq/src/components/pages/Admin/TokenList.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import Loading from 'components/common/Loading/Loading';
+import { useSelector } from 'react-redux';
+import { selectTokens } from 'store/selectedQueue';
 import Token from './Token';
 import styles from './admin.module.scss';
 
-function TokenList({ tokens }) {
-  const Dispaly = () =>
+function TokenList() {
+  const tokens = useSelector(selectTokens);
+  const ListContent = () =>
```

Comment:
Using this selector in admin page was resulting in an unnecessary admin page code re-render, so moved here since only TokenList should be affected by token slice changes.